src/models/index.ts

```diff
@@ -37,7 +37,7 @@ export const DEFAULT_STATE: StateBase = {
   config: DEFAULT_VOYAGER_CONFIG,
   dataset: DEFAULT_DATASET,
   shelf: DEFAULT_SHELF_SPEC,
-  result: DEFAULT_RESULT,
+  result: DEFAULT_RESULT_INDEX,
```

Comment:
`state.result` stores an dictionary (index) for different results.    

`result.main` is for the main result, but we will have other results too.